target/linux/kirkwood/files-5.4/arch/arm/boot/dts/kirkwood-stora.dts

```diff
@@ -0,0 +1,220 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Device Tree file for NETGEAR ReadyNAS NV+ v2
+ *
+ * Copyright (C) 2013, Arnaud EBALARD <arno@natisbad.org>
+ * Copyright (C) 2021, Zoltan HERPAI <wigyori@uid0.hu>
+ */
+
+/dts-v1/;
+
+#include "kirkwood.dtsi"
+#include "kirkwood-6281.dtsi"
+
+/ {
+	model = "NETGEAR MS2000 / MS2110 aka Stora";
+	compatible = "netgear,stora", "netgear,ms2000", "netgear,ms2110", "marvell,kirkwood-88f6281", "marvell,kirkwood";
```

Comment:
I'd prefer the specific name here, something like

```
	model = "NETGEAR Stora MS2110";
	compatible = "netgear,stora-ms2110", "marvell,kirkwood-88f6281", "marvell,kirkwood";
```

or

```
	model = "NETGEAR MS2110";
	compatible = "netgear,ms2110", "marvell,kirkwood-88f6281", "marvell,kirkwood";
```

Netgear seems to use the precise name as well.

Reply:
Netgear used MS2000 for the packages not including a disk, and MS2110 for the packages including a disk, so it's hard to make a distinction. As more sources refer to this as MS2110, I'll pick that one.

Reply:
A better option for maintaining consistency with existing kirkwood dts files would be to rename this file to kirkwood-netgear_stora.dts  See the existing dts files for the ReadyNAS Duo (kirkwood-netgear_duo_v2.dts)  and ReadyNAS NV+ (kirkwood-netgear_nv+_v2.dts).